prolog/comment.pl

```diff
@@ -21,12 +40,9 @@
         );
     SS=S.
 
-eol     --> {atom_codes('\n',E)},id(E).
-eol(HS) --> {atom_codes('\n',E)},id(E,HS).
- 
+%% comment/2 construction - non recursive non tokenizing matcher
```

Comment:
Please either follow pldoc if you want these to be part of the documentation (and anything exported by this module should be), or else just use a single `%`. See http://www.swi-prolog.org/pldoc/man?section=pldoc-comments